apps/docs/src/app/core/component-docs/date-picker/examples/date-picker-form-example.component.ts

```diff
@@ -4,23 +4,34 @@ import { FdDate } from '@fundamental-ngx/core';
 
 @Component({
     selector: 'fd-date-picker-form-example',
-    styles: [`
-        .flex-form{
-            display:flex;
-            justify-content: space-between;
-        }
-    `],
     template: `
         <form [formGroup]="customForm" class="flex-form">
             <div>
-                <fd-date-picker formControlName="date"></fd-date-picker><br/>
+
+
+                <div fd-form-item>
+                    <label fd-form-label>Date Picker</label>
+                    <fd-date-picker [state]="'valid'" formControlName="date"></fd-date-picker>
+                    <fd-form-message [type]="'success'">This is valid DatePicker</fd-form-message>
+                    <br/>
+                </div>
+                
                 Touched: {{customForm.controls.date.touched}}<br/>
                 Dirty: {{customForm.controls.date.dirty}}<br/>
                 Valid: {{customForm.controls.date.valid}}<br/>
                 Selected Date: {{ customForm.controls.date.value ? customForm.controls.date.value.toDateString() : 'null' }}
             </div>
+            <br/>
+            <br/>
+
+            <div fd-form-item>
+                <label fd-form-label>Disabled Date Picker</label>
```

Comment:
The disabled date picker still gets the hand on hover

Reply:
Also add :(core)

Reply:
There is added `pointer-events: none` no disabled popovers.